rakefile

```diff
@@ -9,6 +11,56 @@ include PerfResultsHelper
 
 REF_ARCH_STD, REF_ARCH_LARGE = ["S","L"]
 
+# Create a trumped up class to get a Beaker helpers outside of a Beaker run
+class ClassMixedWithDSLHelpers
```

Comment:
any reason we couldn't dump this class in it's own file and require it... I know the rakefile is already a mess and a half, but if it's easy... not making it bigger wouldn't hurt. :)

Reply:
It is a horrible hack, so I kinda wanted to make it as ugly as possible so that someone would suggest something better.

Reply:
I actually agree with Randell's suggestion and I'm not sure that making the Rakefile worse so that someone will suggest something better is the best approach. Could we put this in a separate file and create a spike ticket to investigate a better alternative to the hack?

Reply:
updated